frontend/src/metabase/lib/colors.js

```diff
@@ -57,3 +58,8 @@ export const harmony = [
     '#c1a877',
     '#f95c67',
 ]
+
+export const getRandomColor = (family: Object) => {
```

Comment:
Instead of Object I'd do

```
type ColorFamily = { [name: string]: string };
```

Or if you want to be really fancy

```
type ColorName = string;
type Color = string
type ColorFamily = { [name: ColorName]: Color };

export const getRandomColor = (family: ColorFamily): Color => {
   // $FlowFixMe: Object.values doesn't preserve the type :-/
   const colors: Color[] = Object.values(family)
```